bin/setup

```diff
@@ -1,20 +1,14 @@
 #!/usr/bin/env ruby
-require 'pathname'
-require 'fileutils'
+
+require "pathname"
 
 gem_root = Pathname.new(__dir__).join("..")
-spec_manageiq = gem_root.join("spec/manageiq")
+system(gem_root.join("bin/before_install").to_s)
 
-if ENV.key?("MANAGEIQ_REPO")
-  manageiq_repo = Pathname.new(ENV["MANAGEIQ_REPO"])
-  puts "== Symlinking spec/manageiq to #{manageiq_repo}"
+require gem_root.join("spec/manageiq/lib/manageiq/environment")
+ManageIQ::Environment.manageiq_plugin_setup(gem_root)
 
-  FileUtils.rm_rf(spec_manageiq.expand_path)
-  FileUtils.ln_s(manageiq_repo.expand_path, spec_manageiq.expand_path)
-elsif !spec_manageiq.exist?
-  puts "== Cloning manageiq sample app =="
-  system "git clone https://github.com/ManageIQ/manageiq.git --branch master --depth 1 spec/manageiq"
+if %w[spec:compile spec:javascript spec:jest].include?(ENV["TEST_SUITE"])
+  puts "\n== Updating UI assets =="
+  ManageIQ::Environment.update_ui
 end
```

Comment:
This part is unique for ui classic, if you compare this PR to other GitHub Actions PRs I've made.